src/components/Sponsor.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import schulich from '../assets/schulich.png'; // Import schulich logo
+import HSBC from '../assets/HSBC.png';
+import zoo from '../assets/zoo.png';
+import '../stylesheets/Sponsor.scss';
+import HYLButton from './HYLButton';
+
+const Sponsor = () => (
+  <>
+    <div id="gradient">
```

Comment:
Just a small modification here; could you change "gradient" to the className and make "#sponsors" the id? (This is for routing purposes)